modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -14,6 +14,11 @@ There are two ways to modify these files to add user-defined functions to GSQL:
 
 * Store the files in a GitHub repository, and configure GSQL to read from the repository.
 * Use `GET` and `PUT` commands to download, modify, and store the files locally.
+** The `GET` command requires the xref:tigergraph-server:user-access:access-control-model.adoc#_privileges[`READ_FILE` privilege].
+** The `PUT` command requires the `WRITE_FILE` privilege.
```

Comment:
why the READ_FILE has reference, and WRITE_FILE does not have xref?

Reply:
The reference is to the privileges section since there are no separate individual headings for the two privileges. 
If there were two references, they would point to the same place, so I only added one